docs/source/en/_toctree.yml

```
@@ -492,6 +492,11 @@
       - local: model_doc/trajectory_transformer
         title: Trajectory Transformer
       title: Reinforcement learning models
+    - isExpanded: false
+      sections:
+      - local: model_doc/time_series_trasnformer
```

The error comes from the fact there is a typo here
```suggestion
      - local: model_doc/time_series_transformer
```